### ISSUE TITLE
issues with start vote feature

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/AsyncVoter#68]
```
Feature: Start Ballot
  As a developer
  So that I can start working on a story, bugfix or chore
  I would like to initiate an asynchronous ballot to get an estimate of how difficult the story will be

  Scenario:
    Given that I submit the URL 'https://github.com/AgileVentures/AsyncVoter/issues/4'
    Then the bot should return an id of that new ballot
  # Scenario: 5-option-ballet

  # Scenario: 3-option-ballet
```
this should be called Start Vote - it should test creation of story with name (as well as URL) and comments should be removed

**Comment by Zsuark** [comment]
Yes, as discussed - we always refer to a story as having three components - name, URL and size.

Size should be left out of the creation process - as it's something we set when the voting is complete.

The defacto consensus appears to be that we set it to 0 for un-voted/in-process stories - side note: 0 is used by some teams as a valid size on completed items. Should we modify this.

So, I think we just need to update the feature here as you've just specified.
